hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -268,4 +280,29 @@ private void verifySnapshotInfoForSnapDiff(final SnapshotInfo fromSnapshot,
           " should be older than to toSnapshot:" + toSnapshot.getName());
     }
   }
+
+  private RocksDB createRocksDbForSnapshotDiff(OzoneConfiguration config) {
+    final Options options = new Options().setCreateIfMissing(true);
+
+    final File dbDirPath =
+        ServerUtils.getDBPath(config, OZONE_OM_SNAPSHOT_DIFF_DB_DIR);
+
+    String dbPath = Paths.get(dbDirPath.toString(), OM_SNAPSHOT_DIFF_DB_NAME)
+        .toFile()
+        .getAbsolutePath();
+
+    try {
+      return RocksDB.open(options, dbPath);
```

Comment:
Please add this `open ` overload to ManagedRocksDB and used it. It'll help you assert if RocksDb objects are not closed properly

Reply:
Added as suggested.